dali/python/nvidia/dali/triton.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+pipeline_autoserialized = False
+
+
+def autoserialize(dali_pipeline):
+    """
+    Decorator, that marks a DALI pipeline (represented by :meth:`nvidia.dali.pipeline_def`) for
+    autoserialization in DALI Backend's model repository.
```

Comment:
I think this doc requires some links and a mention that it's DALI Backend for Triton.

Reply:
Added a link to DALI Backend. I didn't mention Triton here, since this file is named `triton.py`, which make the context obvious. Do you believe I shall mention it also in the doc?

Reply:
I think it would do no harm to include the link. With the docs I think in most cases explicit is better than implicit. Some may not know what Triton is.